pandas/core/indexes/multi.py

```diff
@@ -840,9 +840,44 @@ def size(self) -> int:
 
     @cache_readonly
     def levels(self) -> FrozenList:
-        # Use cache_readonly to ensure that self.get_locs doesn't repeatedly
-        # create new IndexEngine
-        # https://github.com/pandas-dev/pandas/issues/31648
+        """
+        Returns a tuple of Index objects representing the levels
+        of the MultiIndex.
+
+        Each level is an Index object containing unique values
+        from that level of the MultiIndex.
+
+        Returns
```

Comment:
In this case this is an attribute (`MultiIndex.levels` not `MultiIndex.levels()`), so for the users it doesn't return anything, it contains the value, and this section is not expected to exist. I'm not sure how we do for other attributes in the docs, they can be a good reference.